google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -798,6 +805,70 @@ public Map<String, String> extract(ReadModifyWriteRowRequest request) {
         methodName, new ReadModifyWriteRowCallable(retrying, requestContext));
   }
 
+  /**
+   * Creates a callable chain to handle streaming ListChangeStreamPartitions RPCs. The chain will:
+   *
+   * <ul>
+   *   <li>Convert a String format tableId into a {@link
+   *       com.google.bigtable.v2.ListChangeStreamPartitionsRequest} and dispatch the RPC.
+   *   <li>Upon receiving the response stream, it will convert the {@link
+   *       com.google.bigtable.v2.ListChangeStreamPartitionsResponse}s into {@link RowRange}.
+   * </ul>
+   */
+  private ServerStreamingCallable<String, RowRange> createListChangeStreamPartitionsCallable() {
+    ServerStreamingCallable<ListChangeStreamPartitionsRequest, ListChangeStreamPartitionsResponse>
+        base =
+            GrpcRawCallableFactory.createServerStreamingCallable(
+                GrpcCallSettings
+                    .<ListChangeStreamPartitionsRequest, ListChangeStreamPartitionsResponse>
+                        newBuilder()
+                    .setMethodDescriptor(BigtableGrpc.getListChangeStreamPartitionsMethod())
+                    .setParamsExtractor(
+                        new RequestParamsExtractor<ListChangeStreamPartitionsRequest>() {
+                          @Override
+                          public Map<String, String> extract(
+                              ListChangeStreamPartitionsRequest listChangeStreamPartitionsRequest) {
+                            return ImmutableMap.of(
+                                "table_name",
+                                listChangeStreamPartitionsRequest.getTableName(),
+                                "app_profile_id",
+                                listChangeStreamPartitionsRequest.getAppProfileId());
+                          }
+                        })
+                    .build(),
+                settings.listChangeStreamPartitionsSettings().getRetryableCodes());
+
+    ServerStreamingCallable<String, RowRange> userCallable =
+        new ListChangeStreamPartitionsUserCallable(base, requestContext);
+
+    ServerStreamingCallable<String, RowRange> withStatsHeaders =
+        new StatsHeadersServerStreamingCallable<>(userCallable);
+
+    // Copy settings for the middle String -> ByteStringRange callable (as opposed to the inner
```

Comment:
you mean RowRange?

Reply:
I also don't think this comment is accurate, I think you could just say "Copy idle timeout settings for watchdog"